runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropPlugin.java

```diff
@@ -183,4 +198,46 @@ private PrayerType getActivePrayerType()
 		}
 		return null;
 	}
+
+	@Subscribe
+	public void onGameTick(GameTick tick)
+	{
+		if (!config.showFakeXpDrops() || lastSkill == null)
+		{
+			return;
+		}
+
+		tickCounter++;
+
+		if (hasDropped)
+		{
+			hasDropped = false;
+			tickCounter = 0;
+			return;
+		}
+
+		if (tickCounter % config.tickAmount() != 0)
+		{
+			return;
+		}
+
+		clientThread.invokeLater(() ->
+		{
+			client.runScript(2091,  lastSkill.ordinal(), previousExpGained);
```

Comment:
Add `2091` to `ScriptID` with a meaningful name and use it here.

Reply:
Fake experience drop is what Jagex calls it. See: [News](http://services.runescape.com/m=news/upcoming-changes-and-fake-xp-drop-improvements?oldschool=1)

I do however agree additional information is good. Pushing changes.